spec/acceptance/class_spec.rb

```diff
@@ -142,4 +142,29 @@ class { 'grafana':
       it { is_expected.to be_installed }
```

Comment:
It would be nice if we could verify it actually installed a beta release after this. If the repo was not applied correctly, `latest` may still be a stable release.

Reply:
Yes, I had that thought too. However, I couldn't figure out how to adapt this rule such that it would look for the installed version of the Grafana package and validate it to include the substring "beta".

Reply:
Unless you or someone else can suggest code on this topic, I'd resolve this conversation and the PR is ready (again).